pages/api/offer.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import AppAPI from "../../api/AppAPI";
+import MockAPI from "../../api/MockAPI";
+
+let api: AppAPI = new MockAPI();
```

Comment:
```suggestion
let api: AppAPI = new MockAPI();
```

Here we need to use the dependencies injection, maybe I can develop a small library are reinvent the wheel